src/hss_rel14/src/s6as6d_impl.cpp

```diff
@@ -1878,11 +1884,18 @@ AIRProcessor::AIRProcessor(FDMessageRequest& req, s6as6d::Application &app, s6as
      m_app(app),
      m_dict(dict)
 {
+   m_sec = {0};
    m_uimsi = 0;
    m_num_vectors = 0;
+   m_plmn_id[0] = 0;
+   m_plmn_id[1] = 0;
+   m_plmn_id[2] = 0;
+   m_plmn_id[3] = 0;
    m_plmn_len = sizeof(m_plmn_id);
    m_auts_len = sizeof(m_auts);
    m_auts_set = false;
+   memset(m_vector, 0, sizeof(m_vector));
```

Comment:
@lionelgo 

I usually don't like using `memset` to initialize an array.

Do you have an alternative?

Is this class frequently instantiated?

Raphael